### PR TITLE
set is_course_creator variable in cookie

### PR DESCRIPTION
**Pull request by rehan99000**
**Description:**
Sets variable `is_course_creator` in `edly-user-info` cookie.

**JIRA:**
https://edlyio.atlassian.net/browse/EDLY-1412

### CODE REVIEW

File: openedx/features/edly/utils.py [hunk 1]
```diff
@@ -123,10 +123,27 @@ def create_user_link_with_edly_sub_organization(request, user):
...
+    except EdlySubOrganization.DoesNotExist:
+        edly_sub_org = request.site.edly_sub_org_for_studio
     edly_user_profile, __ = EdlyUserProfile.objects.get_or_create(user=user)
     edly_user_profile.edly_sub_organizations.add(edly_sub_org)
     edly_user_profile.save()
 
     return edly_user_profile
+
+
+def update_course_creator_status(request_user, user, set_creator):
```
Comment (ziafazal) [inline openedx/features/edly/utils.py:137]: @rehan99000 where this method is being called?

File: openedx/features/edly/utils.py [hunk 2]
```diff
@@ -123,10 +123,27 @@ def create_user_link_with_edly_sub_organization(request, user):
...
+    """
+    Updates course creator status of a user.
+    """
+    from course_creators.models import CourseCreator
+    try:
+        course_creator = CourseCreator.objects.get(user=user)
```
Comment (ziafazal) [inline openedx/features/edly/utils.py:143]: Should not we use `get_or_create` here?

File: openedx/features/edly/utils.py (hunk 1 above)
Comment (rehan99000) [reply openedx/features/edly/utils.py:137]: In the update method panel app. Let me reference that PR in this one

File: openedx/features/edly/utils.py (hunk 2 above)
Comment (rehan99000) [reply openedx/features/edly/utils.py:143]: This method was only being called from update method in `UserAccountSerializer`. That's why I did this. I will change it

**Comment by rehan99000** [comment]
This PR relates to the following PRs:
https://github.com/edly-io/edly-panel-frontend/pull/186
https://github.com/edly-io/edly-panel-edx-app/pull/79

File: openedx/features/edly/utils.py [hunk 3]
```diff
@@ -123,10 +123,27 @@ def create_user_link_with_edly_sub_organization(request, user):
...
+    Updates course creator status of a user.
+    """
+    from course_creators.models import CourseCreator
+    try:
+        course_creator, _ = CourseCreator.objects.get_or_create(user=user)
+        course_creator.state = CourseCreator.GRANTED if set_creator else CourseCreator.DENIED
+        course_creator.note = 'Course creator user was updated by panel admin {}'.format(request_user.username)
+        course_creator.admin = request_user
+        course_creator.save()
+    except CourseCreator.DoesNotExist:
```
Comment (ziafazal) [inline openedx/features/edly/utils.py:148]: @rehan99000 no need to have this try/exception after using `get_or_create`

Comment (zubair-arbi) [reply openedx/features/edly/utils.py:148]: @rehan99000 you have missed this feedback. Please address this in your next PR.

Comment (zubair-arbi) [reply openedx/features/edly/utils.py:148]: @rehan99000 has addressed this feedback in new PR https://github.com/edly-io/edx-platform/pull/95/

**Pull request merged**